environment.yml

```diff
@@ -0,0 +1,47 @@
+name: hazen-3.9-miniforge
```

Comment:
What does this file do?

Reply:
Sorry, shouldn't have spread between two places. Bit more context in #142.

Basically, installing some of our libraries (mainly scikit-image) doesnt work on M1 silicon via pip. So instead I made the venv in miniforge (just a smaller version of conda). The environment.yml is the equivalent to our normal requirements.txt.

I specifically made most of it install via pip anyway, so it was easy to see which ones work via pip and which only work via conda.

Reply:
Sorry for being a bit slow here. But doesn't the dockerisation of Hazen make this issue a moot point?

Reply:
Sorry I should have made this comment on the issue when you first raised it.